include/roboteam_ai/control/controllers/PidTwoAxesController.h

```diff
@@ -0,0 +1,26 @@
+//
+// Created by ratoone on 25-09-19.
+//
+
+#ifndef RTT_PIDTWOAXESCONTROLLER_H
+#define RTT_PIDTWOAXESCONTROLLER_H
+
+#include <include/roboteam_ai/control/ControlUtils.h>
+#include "BaseController.h"
+#include "PidController.h"
+
+class PidTwoAxesController : BaseController<Vector2,Vector2>{
```

Comment:
The PidTwoAxesController class and the public functions do not have any documentations. Also it is not clear for me what the Pid means. Does it mean "Pelvic inflammatory disease"? This should also be clear from the documentation or you should use the full name instead of the abbreviation Pid.

Reply:
I disagree with this sentiment to be honest. It is googleable and anyone working on this code is very likely to know what it means (and actually does need to know in order to fully understand/edit the code), so this should be common knowledge.

Reply:
I think it is still better to add class documentation, because of the following reasons:
1. When I search for Pid Two Axes Controller I cannot find anything that directly explains how it works. 
2. Documentation of code is a major issue of the RoboTeam.
3. When you add documentation to code we can quickly using CTRL-Q check how a function works. 

If there is some online source which describes how something works then the URL to this source should be included in the documentation.